src/server/cmd/pachctl/cmd/cmd.go

```diff
@@ -375,14 +371,15 @@ Environment variables:
 		Short: "Print Pachyderm version information.",
 		Long:  "Print Pachyderm version information.",
 		Run: cmdutil.RunFixedArgs(0, func(args []string) (retErr error) {
+			if !raw && output != "" {
```

Comment:
I see that this is consistent with what we have elsewhere, but this flag structure has never made much sense to me. I think two or three flags (`--raw`, `--json`, `--yaml` where the first two are equivalent?) would represent things a bit better.

Reply:
We'd still have similar situations where we have to deal with people passing both `--raw` and `--yaml` (or `--json` and `--yaml`), so I don't think it would be more concise.  I'm open to rethinking how we do this, but a little out-of-scope of this PR since this just maintains the status quo in this regard.

Reply:
I was thinking `--raw` would just be there for legacy support, but that's a good point about checking for `--json` and `--yaml`. It just seems annoying to have to do `--raw --yaml` to get yaml now. Agreed it's out of scope

Reply:
In that case I'd rather just do a breaking change and drop `--raw` entirely, so you would just do `--output=json` or `--output=yaml` and the error case goes away entirely and we don't have to check the consistency of flags.